### ISSUE TITLE
Static build

### ISSUE THREAD

**Issue by sjneph** [alexpreynolds/byte-store#2]
I recommend a static build, at least when you start linking up binaries, even though it often requires extra libs to be installed.

**Comment by alexpreynolds** [comment]
Patched in commit 6ac6cb7

**Comment by alexpreynolds** [comment]
Note that building this on Altius hosts may require `module add glibc` to be able to statically link standard math and C libraries. So:

```
$ module purge
$ module add gcc
$ module add glibc
$ make
...
```
